actions/remove_empty_dirs.py

```diff
@@ -7,11 +7,8 @@
 from rucio_consistency import CEConfiguration, Stats
 from rucio_consistency.xrootd import XRootDClient
 
-from run import CCRun
-from config import ActionConfiguration
-
 
-Version = "1.0"
+Version = "1.1"
```

Comment:
Is this separate versioning of components required? 

I hope we can get rid of this in the future and keep a single app version.